# trajectory security config

xf_impact_offset: 0.24
vx_impact: 2.0

xf_max: 0.34
xf_min: -0.5

yf_max: 0.72
yf_min: 0.32

zf_max: 1.5
zf_min: 1.0

tf_max: 1.2
tf_min: 0.6
